.github/dependabot.yml

```
@@ -99,3 +109,50 @@ updates:
       # See https://github.com/dependabot/dependabot-core/issues/4028
       # And, additionally, it is only for performance testing.
       - dependency-name: "org.bsc.maven:maven-processor-plugin"
+  - package-ecosystem: "docker"
+    registries:
+      - dockerhub
+      - ibm
+    # For dependabot to find Docker files they all should be in the same directory.
+    # Dependabot is picking any files that has `dockerfile` in them.
+    #
+    # This includes only Ryuk container, hence we want it to be checked/updated at the same time when our regular java dependencies are updated
+    directory: "/build/container/"
+    schedule:
+      interval: "weekly"
+      day: "tuesday"
+    assignees: ["yrodiere"]
```

I just need a group in order to more easily match the dependabot branches in CI job configuration:

```suggestion
    assignees: ["yrodiere"]
    groups:
      # This group combines all build containers dependencies.
      build-containers:
        patterns:
          # Include all:
          - "*"
```